ext/civi_event/Civi/Api4/Service/Spec/Provider/EventGetSpecProvider.php

```diff
@@ -48,12 +48,26 @@ public function applies($entity, $action) {
    * @param array $maxField
    * @param \Civi\Api4\Query\Api4SelectQuery $query
    * return string
+   *
+   * @throws \CRM_Core_Exception
    */
   public static function getRemainingParticipants(array $maxField, Api4SelectQuery $query): string {
-    $statuses = \CRM_Event_PseudoConstant::participantStatus(NULL, 'is_counted = 1');
+    $statuses = \CRM_Event_BAO_Participant::buildOptions('status_id', NULL, ['is_counted' => 1]);
```

Comment:
@colemanw this gets us away from the deprecated function (including the patches to buildOptions in this commit) - not sure if you have moved onto preferring something else by now...